diesel/Cargo.toml

```diff
@@ -41,7 +43,7 @@ dotenv = ">=0.8, <0.11"
 quickcheck = "0.4"
 
 [features]
-default = ["with-deprecated", "32-column-tables"]
```

Comment:
Did you mean to remove 32-column-tables from default? It bit me in the behind while porting to the new system as the error message when lacking it isn't really that great.

Reply:
Oh, that's only a left over artefact from local development. I tend to disable the `32-column-tables` feature, because this reduces the time needed to compile of diesel quite significantly. 

(But yes, this should be fixed before merging)